src/components/TheSearchBar.vue

```diff
@@ -95,32 +80,32 @@ function isStateChannelId(query) {
 }
 
 async function isAccountName(query) {
-  return await isNameAvailable(query)
+  if (query.endsWith('.test') || query.endsWith('.chain')) {
```

Comment:
Not related but I would expect this function to be called simply `isName`, I don't see the correlation with account

Reply:
Make sense, fixed